src/xrGame/ui/UIGameTutorial.cpp

```diff
@@ -97,6 +97,7 @@ CUISequencer::CUISequencer()
 {
     m_UIWindow = nullptr;
     m_pStoredInputReceiver = nullptr;
+    m_name = "invalid";
```

Comment:
It's better to keep it `nullptr`. What do you think?

Reply:
I just fogot change this on nullptr ;)
Updated